files/build_templates/kube_cni.10-flannel.conflist

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "cbr0",
```

Comment:
what is cbr0?

Reply:
any update?

Reply:
cbr0 is normally a Linux bridge used for CNI, which we don't need for our use case. No cbr0 Linux bridge is created in our use case. However, kubelet expects to find a file here in this format; otherwise, the node's status shows as `NotReady` because normally there is a CNI (We can still run daemonsets on nodes marked NotReady). For readability purposes, this file with these configurations simulates flannel so that the node's status will show as Ready.  In the public kubernetes repo, you can see kubelet checking for the presence and proper formatting of this file in  `pkg/kubelet/dockershim/network/cni/cni.go` and `vendor/github.com/containernetworking/cni/libcni/conf.go`. If we don't have this file here, kubelet throws the warning "cni config is uninitialized"